react/OrderPlaced.js

```diff
@@ -44,13 +47,16 @@ const queryOptions = {
   name: 'orderGroupQuery',
   options: {
     variables: {
-      orderGroup: '902351339185',
+      orderGroup: '886801542094',
```

Comment:
What is this number? It could be a const with a more explicative name (outside the component class maybe)

Reply:
This is the number that identifies the `orderGroup` generated by `checkout`. I use this to get all the information I need from a certain purchase. When `OrderPlaced` is integrated with Dreamstore this number will come as a variable passed to `OrderPlaced`. This number is only hardcoded for testing purposes

Reply:
Created a new `const` with a new name that should make it clearer :)